src/aphroditeWithRTLExtension.js

```diff
@@ -0,0 +1,19 @@
+import { StyleSheet } from 'aphrodite';
+
+/*
+ * When automatically flipping CSS styles in our interface, instead of determining RTL/LTR context
+ * at the time of the create or resolve call (which is hard to do without either setting the
+ * direction in the global cache or ignoring inline styles entirely), we simply add a new style
+ * definition for the generated class name that is only applied in a [dir="rtl"] context. This
+ * handler adds that extra style definition to the <style /> tag created by Aphrodite.
+ */
+function directionSelectorHandler(selector, baseSelector, generateSubtreeStyles) {
+  if (selector !== '_rtl') {
```

Comment:
This string should be a shared constant, instead of being defined in multiple places.